packages/react-components/react-motions-preview/src/types.ts

```diff
@@ -1,13 +1,19 @@
 export type AtomMotion = { keyframes: Keyframe[] } & KeyframeEffectOptions;
 
 export type PresenceMotion = {
-  enter: AtomMotion;
-  exit: AtomMotion;
+  enter: AtomMotion | AtomMotion[];
+  exit: AtomMotion | AtomMotion[];
```

Comment:
`AtomMotion | AtomMotion[]` occurs 9 times in this diff. While there are benefits to being explicit and literal (it's easy to read), DRYing it up in a type could help ensure future APIs support both the singular and plural.

Reply:
I will keep it as it is for now as it's more explicit. We can reconsider this later, it won't be a breaking change.